src/runtime/security/meta_access_controller.cpp

```diff
@@ -65,14 +65,10 @@ meta_access_controller::meta_access_controller(
                                         "RPC_CM_NOTIFY_STOP_SPLIT",
                                         "RPC_CM_QUERY_CHILD_STATE",
                                         "RPC_NEGOTIATION",
-                                        "RPC_CALL_RAW_MESSAGE",
                                         "RPC_CALL_RAW_SESSION_DISCONNECT",
-                                        "RPC_NFS_GET_FILE_SIZE",
```

Comment:
How about this one? It's not duplicated.

Reply:
the RPC of nfs module is managed by the replica_access_controller, so remove it.